### PR TITLE
add coinbase wallet

### LINKED ISSUE #254: Add support for Coinbase Wallet
Details: https://walletlink.org/#/

We had this feature temporarily, but when walletconnect started getting buggy we axed everything. 

I'd like to get to the point were we can support any wallet that our users are using (Zapper does a good job here) 

image

### PR DESCRIPTION
**Pull request by j-yw**
added coinbase wallet support.

Fixes #254

### CODE REVIEW

File: hardhat/deploymentInfo.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
     "ApeRouter": {
-      "address": "0x3D24345bb7FEE3B0B50d66a58946722BBbB26dFf"
+      "address": "0x42c7FfF180Ce8ef5414400696d3BcdD8A6b9354b"
     },
     "ApeToken": {
       "address": "0x3BDaeB7bbCf7659EDcBb4faf22D34A133FF60784"
     },
     "ApeVaultFactoryBeacon": {
-      "address": "0x2997bACD026E3aE48fa57812bFb0cf61DB53baEd"
+      "address": "0x2cF15647fceB0da112EF32803fA55494a151E32D"
```
Comment (exrhizo) [inline hardhat/deploymentInfo.json:39]: Let's not check this in

Comment (j-yw) [reply hardhat/deploymentInfo.json:39]: I'll revert this part and update the PR.

The hardhat setup script produce these and somehow I overlooked it.

Comment (j-yw) [reply hardhat/deploymentInfo.json]: reverted

**Review by exrhizo — APPROVED** [review]
Reese tested this on his desktop and phone, and it works.

**Pull request merged**